test/json_samples.jl

```diff
@@ -296,8 +296,7 @@ twitter = "{\"results\":[
      \"next_page\":\"?page=2&max_id=1480307926&q=%40twitterapi\",
      \"completed_in\":0.031704,
      \"page\":1,
-     \"query\":\"%40twitterapi\"}
-}"
```

Comment:
As an aside, using triple-quotes (`"""`) would make this file much more readable.  (I suspect it was originally created before triple-quotes were part of the language.)